lib/KIcon/iconDefinitions.js

```diff
@@ -87,7 +87,10 @@ const KolibriIcons = {
   audio: { icon: require('./precompiled-icons/material-icons/audiotrack/baseline.vue').default },
   channel: { icon: require('./precompiled-icons/material-icons/apps/baseline.vue').default },
   document: { icon: require('./precompiled-icons/material-icons/book/baseline.vue').default },
-  exercise: { icon: require('./precompiled-icons/material-icons/assignment/baseline.vue').default },
+  exercise: { 
+    icon: require('./precompiled-icons/material-icons/assignment/baseline.vue').default,
+    rtlFlip: true
```

Comment:
Ahh - so now, if this is `true`, and the language is also rtl, then in the `KIcon` file itself, the class of `'rtl-flip-icon'` will also be true, and the icon will be flipped?

Reply:
Yep!